tests/unit/test_routes/test_resource_retreival.py

```diff
@@ -144,5 +148,27 @@ def test_filters(module_client, module_db):
     for resource in response.json['data']:
         assert (resource.get('category') == "Back End Dev")
 
-    # TODO: Filter by updated_after
-    # (Need to figure out how to manually set last_updated and created_at)
+    # Filter by updated_after
+    filter_time = datetime.now() + timedelta(days=-1)
+
+    #   Given I update all resources to be last updated 1 week ago.
```

Comment:
Thanks for including the comments of the walkthrough for my benefit of running through this, but I think when we go to merge, we don't need these comments. The code should speak for itself unless there's something very unclear about _why_ we would be doing a thing.